Messages/index.js

```diff
@@ -0,0 +1,162 @@
+const { matcher } = require("../Lib");
+
+const buildSimpleHandlers = () => {
+  const mapping = [
+    [
+      ['hi Hrach', 'hey Hrach', 'hello Hrach', 'barev Hrach', 'բարև Հրաչ', 'հելլո Հրաչ'],
+      'բարև'
+    ],
+    [
+      ['hrach sus'],
+      'ԽԻԵՍ ԲՈՉԿԱ ԳԼՈՐՈՒՄ ՎՐԵՍ!!!!'
+    ],
+    [
+      ['kilo', 'kg', 'kilogram', 'կիլո', 'կգ', 'կիլոգրամ'],
+      async (message) => {
+        const orer = [
+          'երեկ ',
+          'էսօր ',
+          'էն օրը '
+        ];
+        const tegher = [
+          'տանը ',
+          'ջիմում '
+        ];
+        const text = orer[Math.floor(Math.random()*orer.length)]+
+          tegher[Math.floor(Math.random()*tegher.length)]+
+          'կշռվեցի '+
+          (Math.floor(Math.random() * (105 - 79) + 79)).toString() + ' կիլո էի';
```

Comment:
@tikohov20 please update the numbers, thank you in advance 
![80fe9a85-d09f-415d-8fce-597b70256d7b](https://user-images.githubusercontent.com/36893642/230498643-6a8b02fe-874b-417c-8048-b2ef1cba1bc3.jpg)

Reply:
Numbers are up to date.